Deindent to Make Image 8 Appear

wjohnson opened the pull request:
Removing some indentation around image 8 that left it stuck inside a code block and not rendering.

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @narayanannamalai, has been notified to review your proposed change.

NarayanAnnamalai commented:
#merge-as-is 
Ready to merge
#sign-off

PRmerger commented:
Only the author of the article can use the sign off comment to add the ready-to-merge label.

NarayanAnnamalai commented:
good to merge

TerryLanfear commented:
@NarayanAnnamalai - 4c74356b41 let me know that there were PRs that seemed to be stalled.  Do you know who your pull request approver is?  Can you message your PR approver to merge?

NarayanAnnamalai commented:
@jimdial and @tysonn  to help with the above question:
 Do you know who your pull request approver is? Can you message your PR approver to merge?

NarayanAnnamalai commented:
I am the author of the document and I already approved the merge, 13 hours ago

NarayanAnnamalai commented:
#merge as is. Approved to merge, changes look good.

tysonn commented:
@NarayanAnnamalai  - The article lists the author this way `authors="narayanannamalai"`. Your actual GitHub account is NarayanAnnamalai. GitHub is case sensitive. That is why you were not able to sign off.

There's also a merge conflict that needs to be resolved before this can be merged. See [instructions on the wiki.](https://microsoft.sharepoint.com/teams/azurecontentguidance/wiki/Pages/Resolve%20a%20local%20merge%20conflict%20yourself.aspx)

NarayanAnnamalai commented:
What should I do now, sent out a spate CR fixing the author name?

From: Tyson Nevil [mailto:notifications@github.com]
Sent: Wednesday, August 10, 2016 9:59 AM
To: Azure/azure-content <azure-content@noreply.github.com>
Cc: Narayan Annamalai <narayan@microsoft.com>; Mention <mention@noreply.github.com>
Subject: Re: [Azure/azure-content] proposing change (#7179)


@NarayanAnnamalai<https://github.com/NarayanAnnamalai> - The article lists the author this way authors="narayanannamalai". Your actual GitHub account is NarayanAnnamalai. GitHub is case sensitive. That is why you were not able to sign off.

There's also a merge conflict that needs to be resolved before this can be merged. See instructions on the wiki.<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fmicrosoft.sharepoint.com%2fteams%2fazurecontentguidance%2fwiki%2fPages%2fResolve%2520a%2520local%2520merge%2520conflict%2520yourself.aspx&data=02%7c01%7cnarayan%40microsoft.com%7ce9f703a0b633477e895808d3c13fb253%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636064451713495086&sdata=YhUCAf3g%2b%2by8LSB1A7rKjHL6gRf8v1A3E633%2by4pBfU%3d>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/Azure/azure-content/pull/7179#issuecomment-238932040>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AKkCe_W2KAQpYq0SSGBNtKwBAOnSmEM1ks5qegNwgaJpZM4Jakta>.

KatieCumming commented:
@NarayanAnnamalai - Hi Narayan, just go ahead and fix the merge conflict. Please see the instructions on the wiki in the link Tyson included above. Thanks, Katie

jimdial commented:
@4c74356b41: There ended up being a merge conflict with this proposal and a separate change made to the author value for the article.  The author value was updated in this article, as was incorporation of your suggestion, but was done in a separate PR.  Sorry for the confusion, but you should see your proposed change on the live page.  Thanks for catching it. 

#please-close

4c74356b41 commented:
its okay as long as the fixes are there ;)

wjohnson committed:
Deindent to Make Image 8 Appear

Removing some indentation around image 8 that left it stuck inside a code block and not rendering.
